docs/fabrication/hand-tools/files.md

```diff
@@ -0,0 +1,13 @@
+# Files
+## Uses for a File
+## File Functions
+### Shaping and Smoothing
+### How to Use Files
+#### Gripping the File
+#### Applying Pressure
+#### Direction of Stroke
```

Comment:
How to use Files doesn't need to have so many sub sections. These sections and just be within the steps of using a file